### ISSUE TITLE
webMAN Refresh XML broken

### ISSUE THREAD

**Issue by illusionalDEX** [aldostools/webMAN-MOD#182]
No matter how i refresh XML it will freeze my console i have tried re-installing webMAN re-installed firmware to make sure it wasnt either one of those and well it wasnt so i dont know if its something to do with webMAN or somethiing else...

**Comment by mogzol** [comment]
Possibly the same issue as #181. Versions of webman newer than 1.47.14 have been freezing for a lot of people (including me). Downgrading to 1.47.14 seems to be the best solution until aldostools can release a fix.

**Comment by aldostools** [comment]
Thank you for your feedback... I'm currently working in the issue.
Maybe I'll need some tests in few minutes.

**Comment by aldostools** [comment]
Sorry for the delay... @Joonie86 has been helping me identifying the issue.  BIG THANKS TO HIM!!
It seems to be related to a new thread created during save settings.

Attached there are 3 test versions... please report if some of them work or if the 3 versions still freeze.

[webMAN-MOD_test_versions.zip](https://github.com/aldostools/webMAN-MOD/files/3106065/webMAN-MOD_test_versions.zip)

**Comment by shae23** [comment]
.17 test 1 resolves the issue on my end

**Comment by aldostools** [comment]
Thank you for your feedback!

I want to thanks again to @Joonie86 for his patience and great tips that helped me to find the issue.

I have uploaded the fixed version to Brewology and in [releases](https://github.com/aldostools/webMAN-MOD/releases) section of Github. 
Removed metalification and rebugification themes until I release .18 + src  ;)

My most sincere apologies for all the inconveniences that this bug had caused.
